.github/workflows/build.yaml

```diff
@@ -137,13 +138,32 @@ jobs:
           path: /tmp/artifacts/
           name: packages-aarch64
 
-      - run: echo "${{ secrets.MELANGE_RSA }}" > ./wolfi-signing.rsa
-      - run: |
-          mkdir -p /etc/apk/keys
-          cp ./wolfi-signing.rsa.pub /etc/apk/keys/wolfi-signing.rsa.pub
+      - name: 'Upload APKs to the bucket'
+        if: always()
+        run: |
+          for arch in "x86_64" "aarch64"; do
+          mkdir -p ./packages/${arch}
+
+          # Consolidate with the built artifacts
+          tar xvf /tmp/artifacts/packages-${arch}.tar.gz
+
+            # Upload any APKs we were able to build.
+            apks=$(ls ./packages/${arch}/*.apk 2>/dev/null)
+            if [ -n "$apks" ]; then
```

Comment:
I'm worried about uploading APKs that aren't present for all archs because I believe if we withdraw in that state the index will be recomputed and we'll end up in an inconsistent state?

Reply:
I think we also need to deal with some cases where APKs only exist for a single arch in enterprise too, so IDK how we reconcile those 🙃